### DISCUSSION TITLE
Different volumes for Speaker vs. Headphones

### DISCUSSION THREAD

**Discussion by theUnBurn** [OnionUI/Onion#1591] [Ideas]
I frequently find myself using headphones with the volume high, then later I turn on my Miyoo again and the speakers are blasting.

Would love a way to expose which output the sound is coming out of and remember the last used volume for a given output (or mute it when headphones are plugged out).

**Comment by DaRkViVi** [comment]
I literally logged in to ask the same. I don’t know if it’s possible, but in any case +1 for me.
